static/app/views/settings/project/server-side-sampling/modals/recommendedStepsModal.spec.tsx

```diff
@@ -7,37 +9,59 @@ import {RecommendedStepsModal} from 'sentry/views/settings/project/server-side-s
 import {SERVER_SIDE_SAMPLING_DOC_LINK} from 'sentry/views/settings/project/server-side-sampling/utils';
 
 import {
-  getMockData,
+  getMockInitializeOrg,
   mockedProjects,
+  mockedSamplingDistribution,
   mockedSamplingSdkVersions,
   recommendedSdkUpgrades,
   uniformRule,
 } from '../testUtils';
 
+function ComponentProviders({children}: {children: React.ReactNode}) {
```

Comment:
I assume that when we go ahead with react-query we'll put this provider in with all the others so this won't be necessary to do in each test file?

For tests we'd also want to pass some different options to QueryClient to make tests run faster like disabling retries:

```ts
const queryClient = new QueryClient({
  defaultOptions: {
    queries: {
      retry: false
    },
    mutations: {
      retry: false
    }
  },
})
```

Reply:
> I assume that when we go ahead with react-query we'll put this provider in with all the others so this won't be necessary to do in each test file?

I saw that you worked on this in the PR https://github.com/getsentry/sentry/pull/40290 but since some folks were not in favor of it a while ago, I think we should discuss this again in the Frontend TSC

Reply:
> For tests we'd also want to pass some different options to QueryClient to make tests run faster like disabling retries:

oh, yes! definitely